agent/src/main/java/com/appland/appmap/record/Recorder.java

```diff
@@ -1,18 +1,18 @@
 package com.appland.appmap.record;
 
-import java.io.IOException;
```

Comment:
It doesn't look like most of these changes to the imports are necessary. Can you limit them to what's required for the new functionality, please?

Reply:
Oh, sorry about that. Default IDE settings for new project and I missed it. Fixed in the new version